src/packages/vite-plugin-graphweaver/src/loaders/custom-fields.ts

```diff
@@ -0,0 +1,25 @@
+import path from 'path';
+import fs from 'fs/promises';
+
+export const loadCustomFields = async (configPath: string) => {
+	try {
+		let customFieldsPath = path.resolve(process.cwd(), 'src', 'admin-ui', 'custom-fields');
+
+		try {
+			const { adminUI } = await import(configPath);
+			if (adminUI?.customFieldsPath) customFieldsPath = adminUI.customFieldsPath;
```

Comment:
Can these just be another export from the existing admin UI file? Or is there a technical reason that doesn't work